dapper/mods/L96s/__init__.py

```diff
@@ -0,0 +1,119 @@
+########################################################################################
+########################################################################################
+"""A 1D emulator of chaotic atmospheric behaviour, including additive noise.
+
+`bib.grudzien2020numerical`
+"""
+
+
+########################################################################################
+########################################################################################
+# Imports
+import numpy as np
+
+from dapper.mods.integration import rk4
+
+from .extras import LPs, d2x_dtdx, dstep_dx
+
+########################################################################################
+########################################################################################
+# Method definitions
+########################################################################################
+########################################################################################
+# basic Lorenz 96 defintions and functions
+
+__pdoc__ = {"demo": False}
+
+# energy injected into the system
+Force = 8.0
+
+# diffusion in the SDE process
+s = 0.05
+
+# ?
+Tplot = 10
+
+
+def x0(M):
+    return np.eye(M)[0]
+
+
+def shift(x, n):
+    return np.roll(x, -n, axis=-1)
+
+
+def dxdt_autonomous(x):
+    return (shift(x, 1)-shift(x, -2))*shift(x, -1) - x
+
+
+def dxdt(x):
+    return dxdt_autonomous(x) + Force
+
+
+def step(x0, t, dt):
+    return rk4(lambda t, x: dxdt(x), x0, np.nan, dt)
+
+
+########################################################################################
+# 2nd order strong taylor SDE step
+
+def l96s_tay2_step(x, t, dt):
```

Comment:
The main reason for using this scheme is for the high-accuracy of the truth twin simulation, where this scheme is only designed for the Lorenz-96 model with additive noise that has a scalar covariance at all times.   The key is that strong convergence needs to be the criterion for whether a simulation of the truth twin will be consistent with the equations of motion in generating an observation process in a perfect-random model.  Because this is a specific sub-class of stochastic Lorenz-96 models (though one that is probably most commonly used in the SDE configuration) I decided to name it the L96-s model where the s refers to the stochasticity, and particularly the diffusion coefficient which defines an "instantaneous" standard deviation of the noise process.

Reply:
Got you, I think. You still might be able to avoid duplicating `Lorenz96.py/extras.py` and large parts of `__init__.py` by importing them from `Lorenz96` into `L96s`, I belive.